install.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Then why the installation instructions of TiDB or Rocksdb are included in TiKV project?